spec/index.bs

```diff
@@ -2462,6 +2484,28 @@ texels, used when copying data from or to a {{GPUTexture}}.
 
   * {{GPUTextureCopyView/origin}}: If unspecified, defaults to `[0, 0, 0]`.
 
+<div algorithm class=validusage>
+
+<dfn abstract-op>GPUTextureCopyView Valid Usage</dfn>
+
+  Given a {{GPUTextureCopyView}} |textureCopyView|, let
+  - |blockWidth| be the [=texel block width=] of |textureCopyView|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+  - |blockHeight| be the [=texel block height=] of |textureCopyView|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+
+  The following validation rules apply:
+  - |textureCopyView|.{{GPUTextureCopyView/texture}} must be a [=valid=] {{GPUTexture}}.
+  - The {{GPUTexture/[[sampleCount]]}} of |textureCopyView|.{{GPUTextureCopyView/texture}} must be 1.
+  - |textureCopyView|.{{GPUTextureCopyView/mipLevel}} must be less than the {{GPUTexture/[[mipLevelCount]]}} of
+    |textureCopyView|.{{GPUTextureCopyView/texture}}.
+  - If the |textureCopyView|.{{GPUTextureCopyView/texture}} is {{GPUTextureDimension/1d}} or {{GPUTextureDimension/3d}}:
+    - |textureCopyView|.{{GPUTextureCopyView/arrayLayer}} must be zero.
+  - If the {{GPUTexture/[[dimension]]}} of |textureCopyView|.{{GPUTextureCopyView/texture}} is {{GPUTextureDimension/2d}}:
+    - |textureCopyView|.{{GPUTextureCopyView/arrayLayer}} must be less than the {{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=] of |textureCopyView|.{{GPUTextureCopyView/texture}}.
+  - |textureCopyView|.{{GPUTextureCopyView/origin}}.{{GPUOrigin3DDict/x}} must be a multiple of |blockWidth|.
+  - |textureCopyView|.{{GPUTextureCopyView/origin}}.{{GPUOrigin3DDict/y}} must be a multiple of |blockHeight|.
```

Comment:
These rules are the same for both buffer-to-texture and texture-to-buffer copies so I decide to put it here.